WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1537,6 +1537,25 @@ private boolean addMediaVisualEditor(Uri uri, boolean isVideo) {
             return false;
         }
 
+        if (!isVideo && SiteSettingsInterface.getInterface(this, mSite, null).init(false).getOptimizedImage() &&
+               !NetworkUtils.isWiFiConnected(this)) {
+            // Not on WiFi and optimize image is set to ON
+            String optimizedPath = ImageUtils.optimizeImage(this, path);
+
+            if (optimizedPath == null) {
+                AppLog.e(T.EDITOR, "Optimized picture was null!");
+                //TODO: track analytics here
```

Comment:
minor, very minor comment: please use space after the `//`. Like, `// TODO` instead of `//TODO`. I had maxime ask me for this once 😄